src/psyclone/tests/psyir/frontend/fparser2_test.py

```diff
@@ -636,19 +636,22 @@ def test_process_declarations():
             Symbol.Visibility.PRIVATE)
 
     # Initialisations of static constant values (parameters)
-    reader = FortranStringReader("integer, parameter :: i1 = 1")
+    reader = FortranStringReader("integer, private, parameter :: i1 = 1")
```

Comment:
Is this private attribute and the two visibility asserts added here needed. Since now these are properly tested somewhere else I think it will be more unit-testy without it.

Reply:
OK, removed.